SteamList/GamesViewController.swift

```diff
@@ -26,23 +38,37 @@ class GamesViewController: UIViewController {
 extension GamesViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return numberOfGames
+        return games.count
     }
     
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for:  indexPath)
-        cell.textLabel?.text = "Game \(indexPath.row)"
-        cell.textLabel?.textColor = .white
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellId, for:  indexPath) as! GameTableViewCell
+        let game = games[indexPath.row]
+        cell.cellTitle.text = "\(game.title)"
+        if game.isFavorite {
+            cell.cellIcon.imageView?.image = UIImage(systemName: "star.fill")
+        } else {
+            cell.cellIcon.imageView?.image = UIImage(systemName: "star")
+        }
+        
         return cell
     }
     
-
-    
-    
     
 }
 
 extension GamesViewController: UITableViewDelegate {
     
 }
+
+
+struct Game {
```

Comment:
I would suggest to create a different swift file for the model

Reply:
done